src/server/config.js

```diff
@@ -56,6 +56,9 @@ export default function (baseConfig, configDir) {
     module: {
       ...config.module,
       // We need to use our and custom loaders.
+      preLoaders: [
```

Comment:
Actually, there's a much better generic way to deal with this.
Try this instead of this one. I hope it'll work.

```js
...customConfig.module,
```

If this works, send me a PR with that.